Signal/src/call/PeerConnectionClient.swift

```diff
@@ -95,7 +119,7 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     // Connection
 
-    private var peerConnection: RTCPeerConnection!
+    private var peerConnection: RTCPeerConnection?
```

Comment:
`peerConnection` is cleared after `terminateInternal`, so this needs to be optional.